### PR TITLE
Skeleton for mobile version

### PR DESCRIPTION
**Pull request by mullenlucas**
This is the first commit for the 2 first sections of the portfolio project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio Project</title>
+</head>
+<body>
+    <nav class="nav-bar">
+        <h1 class="title">Welcome!</h1>
+        <a class="icon">
+            <i class="fa fa-bars"></i>
+        </a>
+    </nav>
```
Comment (codecaiine) [inline index.html:17]: - [ ] Good job with your toolbar, however, there are some issues you need to address, please put all your toolbar elements in a `<header>` tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio Project</title>
+</head>
+<body>
+    <nav class="nav-bar">
+        <h1 class="title">Welcome!</h1>
```
Comment (codecaiine) [inline index.html:13]: - [ ] Please add a link to your logo by using  `<a>`. the tags `<h1>` to `<h4> should be used to organize information and orient users, the logo is not actually information or orientation.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        <a class="icon">
+            <i class="fa fa-bars"></i>
+        </a>
+    </nav>
+    <section class="headline">
+        <div class="intro-txt">
+            <h2 class="wave-txt">Hey there. I'm Lucas</h2>
```
Comment (codecaiine) [inline index.html:20]: - [ ] Please use a `<h1>` tag for this main title.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            my first project using flexbox! Below are the social media links to contact me
+            </p>
+        </div>
+        <div class="media-imgs">
+            <img src="imgs/twitter.svg" alt="Twitter img">
+            <img src="imgs/Linkedin-icon.svg" alt="Linkedin img">
+            <img src="imgs/medium.svg" alt="Medium img">
+            <img src="imgs/github.svg" alt="Github img">
+            <img src="imgs/angellist.svg" alt="Angellist img">
+        </div>
```
Comment (codecaiine) [inline index.html:35]: - [ ] Well done, you added all social media Icons but please use the `ul` tag to group your social media icons, and `li` per icon, and put each icon inside a link ` a.

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,72 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500&display=swap');
+
+/* Mobile setup */
+@media (min-width: 400px) {
```
Comment (codecaiine) [inline styles.css:4]: - [ ] Good job with the media query but your minimum mobile screen size should be `375`px and make sure that mobile version should be responsive until the desktop screen size.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+        <div class="main-headline-txt">
+            <p>
+            I am begginer student at Microverse, currently learning how to implement flexbox properties in CSS. This is
+            my first project using flexbox! Below are the social media links to contact me
+            </p>
+        </div>
+        <div class="media-imgs">
+            <img src="imgs/twitter.svg" alt="Twitter img">
+            <img src="imgs/Linkedin-icon.svg" alt="Linkedin img">
```
Comment (codecaiine) [inline index.html:31]: - [ ] Your LinkedIn icon is broken , please add the correct path for it in order to fix that.

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @mullenlucas , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
:heavy_check_mark: Commits meaningful
:heavy_check_mark: No linters errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mullenlucas) [reply index.html]: Thank you, I wrapped the nav inside a header

File: styles.css (hunk 5 above)
Comment (mullenlucas) [reply styles.css]: I seem to have confused the max with the min. I would've caught that if I had tested it in mobile version on the browser inspect tool. Now I added min-width to 375px and max-width to 600px. On those ranges, the website acquires the styling.

**Comment by mullenlucas** [comment]
The website acquires the styling between 375px an 600px ranges

File: index.html (hunk 6 above)
Comment (mullenlucas) [reply index.html]: Somehow on my last linter review previous to submitting, this didn't came up. Now it is corrected, thank you !

**Comment by mullenlucas** [comment]
Sorry for unmeaningful commit messages. I was trying to correct the linter error over and over again and wasn't doing much changes other than trying to pass the linters. Then I learned how to test for those linter errors locally before unnecessarily commiting to github

File: index.html [hunk 7]
```diff
@@ -9,30 +9,38 @@
     <title>Portfolio Project</title>
 </head>
 <body>
+    <header>
     <nav class="nav-bar">
+        <a href="#">
         <h1 class="title">Welcome!</h1>
```
Comment (mirouhml) [inline index.html:15]: - [ ] Please know that as the previous reviewer mentioned, the `hx` tags should be used to organize information and orient users the logo isn't one of those, kindly change that. 😁
- [ ] Also when you added the `a` tag the styling changed for the logo, kindly fix that.

Currently it's like this in your website:
image
But according to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) it should be like this:
image

File: index.html [hunk 8]
```diff
@@ -9,30 +9,38 @@
...
     </nav>
-    <section class="headline">
+    </header>
+    <section class="main-container">
+    <div class="headline">
         <div class="intro-txt">
-            <h2 class="wave-txt">Hey there. I'm Lucas</h2>
-            <h2 class="desc-txt">I'm a software developer</h2>
+            <h1 class="wave-txt">Hey there. I'm Lucas</h1>
+            <h1 class="desc-txt">I'm a software developer</h1>
```
Comment (mirouhml) [inline index.html:26]: - [ ] Please know that a website can't have more than one `h1` tag (this only applies for `h1` tag not the rest of the `hx` tags), kindly fix that 😁

File: index.html [hunk 9]
```diff
@@ -9,30 +9,38 @@
...
-            <img src="imgs/medium.svg" alt="Medium img">
-            <img src="imgs/github.svg" alt="Github img">
-            <img src="imgs/angellist.svg" alt="Angellist img">
+            <ul>
+            <li><img src="imgs/twitter.svg" alt="Twitter img"></li>
+            <li><img src="imgs/Linkedin-icon.svg" alt="Linkedin img"></li>
+            <li><img src="imgs/medium.svg" alt="Medium img"></li>
+            <li><img src="imgs/github.svg" alt="Github img"></li>
+            <li><img src="imgs/angellist.svg" alt="Angellist img"></li>
+            </ul>
```
Comment (mirouhml) [inline index.html:41]: [OPTIONAL]

- [ ] I suggest that you fix the indentation here, as it will make your code organized and easy to read.

**Review by mirouhml — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet, but you're almost there!

image


 ## Highlights
- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 7 above)
Comment (mullenlucas) [reply index.html:15]: I removed the <h1> tag, hadn't understood that previously. I didn't know which tag to use for the logo-text, so I used the <p> tag, hope that is correct.
Also, used a tiny bit of a "return false" on the onClick parameter on the <a> tag (a trick I found online), so I could make as though the logo is a link but doesn't direct to anywhere (so when hovering over with the mouse it functions as a link and not as a text)

File: index.html (hunk 8 above)
Comment (mullenlucas) [reply index.html:26]: Great suggestion ! Didn't know this, thanks

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @mullenlucas  🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great work on making the changes requested by a previous reviewer 👏
- Your repo is professional ✅
- Well structured files ✅

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**